vc_infer_pipeline.py

```diff
@@ -50,6 +51,24 @@ def change_rms(data1, sr1, data2, sr2, rate):  # 1是输入音频，2是输出
     return data2
 
 
+class F0Type(str, Enum):
+    Hz = "f0_hz"
+    Log = "f0_log"
+    Mel = "f0_mel"
+
+
+def hz2mel(f0):
+    return 1127 * np.log(1 + f0 / 700)
+
+
+def mel2hz(mel):
+    return 700 * (np.exp(mel / 1127) - 1)
+
+
+def filter_out_silence(f0):
+    return np.array(list(filter(lambda x: x > 60, f0)))

```

Comment:
will be much faster to work in numpy, not very important as the list is relatively small

Reply:
are you filtering by the frequency?
to filter silence you need to filter by magnitude
(also need to filter by frequency to eliminate gross errors)

Reply:
All I get at this point is the frequency.
Again - I do not want to change the RVC code (and filtering out silence will require changing that code).
Maybe "filter_out_silence" is not a good name, but silence ~= white noise which will have very low frequency, so I think it's comparable.

Reply:
How do I do such filtering in numpy?

Reply:
Changed to `band_filter`.